### ISSUE TITLE
Is there a sample project of this library in use? Examples of how to validate receipts on iOS?

### ISSUE THREAD

**Issue by Nerves82** [jamesmontemagno/InAppBillingPlugin#280]
I am in the process of implementing Receipt Validation with Xamarin Forms iOS using this library and having trouble understanding what I am actually suppose to do with the "signedData" string in the VerifyPurchase callback. When I decode it I get a mix of gibberish and some actual information. I can find plenty of information on validation when using iOS native or Xamarin Native, but almost nothing regarding this library. The android side gets more technical information then the iOS side dose so at this point I am just kinda stuck. Can someone point me to some sample code please? Thanks.

**Comment by jamesmontemagno** [comment]
See the docs -> https://jamesmontemagno.github.io/InAppBillingPlugin/SecuringPurchases.html

**Comment by Nerves82** [comment]
The docs say nothing about the proper way to decode the receipt. The docs also say noting about on device validation, only server side validation using azure functions. I have tried every encoding I can find and the receipt is always part decoded and part gibberish. I pulled the entire plugin directly into my project and checked out the receipt that is given from the NSMainBundle and it too is gibberish, so at this point I am completely lost and am finding hardly any help anywhere online.
